__tests__/Pipeline.integration.js

```diff
@@ -128,6 +131,101 @@ describe('integration tests', () => {
     }
   })
 
+  test('Trellis resources are re-indexed', async () => {
+    // if not already present, create the base repository container as we'd have in practice, so there's some structure to crawl
+    await superagent.head(`${config.get('platformUrl')}/repository`)
+      .catch(async _err => {
+        await superagent.post(`${config.get('platformUrl')}`)
+          .type('application/ld+json')
+          .send('{ "@context": { "rdfs": "http://www.w3.org/2000/01/rdf-schema#", "ldp": "http://www.w3.org/ns/ldp#" }, "@id": "", "@type": [ "ldp:Container", "ldp:BasicContainer" ], "rdfs:label": "Sinopia LDP Server" }')
+          .set('Link', '<http://www.w3.org/ns/ldp#BasicContainer>; rel="type"')
+          .set('Slug', 'repository')
+          .then(res => res.body)
+          .catch(err => { console.dir(err); throw err })
+      })
+
+    const resourceCount = 5
+    await Promise.all([...Array(resourceCount).keys()].map(i =>
+      superagent.post(`${config.get('platformUrl')}/repository`)
+        .type('application/ld+json')
+        .send(`{ "@context": { "mainTitle": { "@id": "http://id.loc.gov/ontologies/bibframe/mainTitle" } }, "@id": "", "mainTitle": [{ "@value": "${reindexingResourceTitle} ${i}", "@language": "en" }] }`)
+        .set('Link', '<http://www.w3.org/ns/ldp#BasicContainer>; rel="type"')
+        .set('Slug', `${reindexingResourceSlug}_${i}`)
+        .then(res => res.body)
+        .catch(err => { console.dir(err); throw err })
+    ))
+
+    // Give the pipeline a chance to run
+    await sleep(4900)
+
+    await Promise.all([...Array(resourceCount).keys()].map(i => {
+      const identifier = `repository/${reindexingResourceSlug}_${i}`
+      return client.search({
+        index: config.get('resourceIndexName'),
+        type: config.get('indexType'),
+        body: {
+          query: {
+            term: {
+              _id: {
+                value: identifier
+              }
+            }
+          }
+        }
+      }).then(response => {
+        // including phrase makes it easier to find the one that fails the test, should the test fail
+        expect([identifier, response.hits.total]).toEqual([identifier, response.hits.total])
```

Comment:
@jmartin-sul I didn't realize until just now that I should have asked: can this test (or the one on line 224) ever fail?

Reply:
d'oh!  good catch @mjgiarlo.  i believe the right hand side of the comparison in both cases should be `1` (since it's doing a specific search for the constructed slug, ```${reindexingResourceSlug}_${i}```, so as not to get thrown off by other things in thing index).

um, file a bug, i guess?  should be a pretty quick fix, unless turning these lines into real tests reveals a bug, but hopefully that won't be the case!

Reply:
I'm already in code nearby so I'll have at it. thank you!

Reply:
nice, np, thank you for spotting that!